### ISSUE TITLE
Flaps and Spoilers Issue

### ISSUE THREAD

**Issue by Modsojoy** [Alexustas/Props-requests#42]
this is either a CFG Error from your pack or an error from JSI, It seems to be taking issue with the JSIFAR Variable. - 
![20160828022800_1](https://cloud.githubusercontent.com/assets/21351692/18120295/db565220-6f2c-11e6-93e0-b99042d9f32f.jpg)

[KSP.txt](https://github.com/Alexustas/Props-requests/files/446651/KSP.txt)

**Comment by Modsojoy** [comment]
Im using your FAR Spoilers and flaps levers in my IVA.

**Comment by Alexustas** [comment]
I just checked again that props and everything works fine
You [need to add ](https://github.com/Mihara/RasterPropMonitor/wiki/Configuring-a-monitor#rasterpropmonitorcomputer-configuration) in the config of your cabin (Part) following module

```
MODULE
{
    name = RasterPropMonitorComputer
}
```

**Comment by Modsojoy** [comment]
that does not make any sense, Its ONLY The Levers, although i am using all Retro/Analogue Props Minus the SAS Button, Alright.

**Comment by Modsojoy** [comment]
To the pod or the IVA Cfg?

**Comment by Alexustas** [comment]
> To the pod or the IVA Cfg?

Pod (part) cfg

**Comment by Alexustas** [comment]
and make sure you are using the most [recent version of props](https://www.dropbox.com/s/evzc2zos8g9iecv/ASET_Dev-24-08-2016.zip?dl=0)

**Comment by Modsojoy** [comment]
wait you have been updating as late as the 24th, But the thread has been dead in the water. ANYWAYS, Was unaware that there were Updates. im using one that has some Extra bits, the last release from the Forum thread but recompiled for 1.1.3 Anything been removed?

**Comment by Modsojoy** [comment]
do the nav radios do anything?

**Comment by Alexustas** [comment]
> do the nav radios do anything?

Yes of course. VOR, ILS and ADF navigation available

Charts:
https://www.dropbox.com/s/c64mhqxpnhldze7/Kerbin_AEROCHARTS-24aug2016.pdf?dl=0

**Comment by Modsojoy** [comment]
So i can fly ILS Into R09L. FUCKY YEAH!

**Comment by Alexustas** [comment]
> the last release from the Forum thread but recompiled for 1.1.3 

I can not provide any support to that published unofficially
the new release will be released as soon as the beta-tests finished, do not think more than a week

**Comment by Modsojoy** [comment]
Are you going to remove the Requested specialty props/MOARdv's bits or just separate them into a .zip within a .zip like you did with the FDAI

**Comment by Alexustas** [comment]
> So i can fly ILS Into R09L. FUCKY YEAH!

yes, when the _second_ runway will be built ))

**Comment by Modsojoy** [comment]
Er - Just 09, Whatevs, Im going to try and find the stock runway textures and make an 09L Static for KerbalKonstructs

**Comment by Alexustas** [comment]
> Are you going to remove the Requested specialty props/MOARdv's 

yes

**Comment by Modsojoy** [comment]
damnit, so they wont be supported anymore? as in no longer updated

**Comment by Alexustas** [comment]
what are you talking about?

**Comment by Modsojoy** [comment]
"Are you going to remove the Requested specialty props/MOARdv's"
"yes"
So are they going to be maintained by anyone or by removed you mean your going to delete them entirely.

**Comment by Alexustas** [comment]
These props were created specifically for the request by MOARdv and are not used in "ASET" internals at all, but are used in "MOARdv" internals, I made a model only and the most simple, basic configuration, nothing more.
so I think it is logical to include these props in MOARdv's release

**Comment by Modsojoy** [comment]
Didnt know MOARdv was continuing his FlightSystemsRedux.

**Comment by Alexustas** [comment]
you can take this models and use with your configs as you like and include them in your releases too

**Comment by Alexustas** [comment]
> Didnt know MOARdv was continuing his FlightSystemsRedux.

Yes, he continues to work on his IVA, as far as I know

**Comment by Modsojoy** [comment]
The props under the MOARdv folder is what i was talking about, especially the FDAI

**Comment by Modsojoy** [comment]
Also - do you have any Props laying around that can Lock resources? Like a Covered switch that when Uncovered and flipped Unlocks ElectricCharge, because im doing a 737Esq cockpit and need something like that for the overhead.

**Comment by Alexustas** [comment]
Oh, do not pay attention to "FDAI.zip" (in the _ASET/ASET_Props/MOARdv_), I just forgot to remove it)) 
FDAI device included in the "ASET Props" pack (_ASET/ASET_Props/Instruments/FDAI_)

**Comment by Alexustas** [comment]
> Also - do you have any Props laying around that can Lock resources? 

No, RPM allows only **un**lock resources (all at once) that were previously blocked

**Comment by Modsojoy** [comment]
No way to create a custom Defined Variable?

**Comment by Modsojoy** [comment]
Because i have seen something that can lock (all resources) from IVA. just dont know where.

**Comment by Alexustas** [comment]
you need to carefully read the [RPM-Wiki](https://github.com/Mihara/RasterPropMonitor/wiki) for more information

**Comment by Modsojoy** [comment]
I cant do anything worth of coding. i can do about 2 lines of lua, and thats with variables n shit already defined somewhere

**Comment by DeputyLOL** [comment]
That OK because RPM doesn't use lua

**Comment by Modsojoy** [comment]
thats worse, i only know Lua!

**Comment by Modsojoy** [comment]
and atleast a page of coding would be needed probably to get the function i need for this one. god damned. switch.

**Comment by Modsojoy** [comment]
Ran into an error with the Electric output gauge from the avionics classic bits
"Cannot clone model 'ASET/ASET_Avionics/ClassicPack/ASET_Elec_Output/ASET_Elec_Output' as model does not exist"

**Comment by Alexustas** [comment]
> thats worse, i only know Lua!

I would advise you to pay attention to a [completely new system](https://github.com/MOARdV/AvionicsSystems), which MOARdv developed as a replacement for RPM

**Comment by Modsojoy** [comment]
Oh wait, i know whats wrong
Nvm

**Comment by Modsojoy** [comment]
ok, solve one and created another, the break force gauge is coming up in place of the Electric output gauge

**Comment by Alexustas** [comment]
> Ran into an error with the Electric output gauge

Yes, this error occurs. I have corrected it, but has not yet updated dev-build. This error does not produce any effect, only displays a message in the debug menu

**Comment by Alexustas** [comment]
> Yes, this error occurs. I have corrected it, 

oops, I messed up this error with another very similar error

**Comment by Modsojoy** [comment]
Gotta say though, these props are beautiful

**Comment by Modsojoy** [comment]
How do i know whats going to appear on the SquareMonitors type A And B.

**Comment by Alexustas** [comment]
> "Cannot clone model 'ASET/ASET_Avionics/ClassicPack/ASET_Elec_Output/ASET_Elec_Output' as model does not exist"

I just checked, props you specified load properly without errors

**Comment by Modsojoy** [comment]
It loaded propery Sorta, in place of the electric output gauge it spawned a Break force gauge

**Comment by Alexustas** [comment]
> It loaded propery Sorta, in place of the electric output gauge it spawned a Break force gauge

I mean, I have no  errors that you have specified in my log, and both props displayed correctly in the game

**Comment by Modsojoy** [comment]
Hm.

**Comment by Modsojoy** [comment]
Well no matter. i decided to scrap that bit.

**Comment by Alexustas** [comment]
> How do i know whats going to appear on the SquareMonitors type A And B.

```
<<TESTTESTTESTTESTTESTTESTTESTTESTTEST>>
TEST
TEST
TEST
TEST
TEST
TEST
TEST
TEST
TEST
TEST
TEST
```

**Comment by Modsojoy** [comment]
... So they are not usable?

**Comment by Alexustas** [comment]
> ... So they are not usable?

as I said earlier:

> I made a model only and the most simple, basic configuration, nothing more.

**Comment by MOARdV** [comment]
> ... So they are not usable?

As I posted in another issue - you need to write your own configs for those monitors.  alexustas made the prop for me so I could use them.

... Which it looks like he also just now replied.

**Comment by Modsojoy** [comment]
Alright both Chill.
